.github/workflows/integration-test.yml

```diff
@@ -97,61 +72,60 @@ jobs:
 
       - name: setup-dfx
         run: |
-          mkdir -p ~/.local/bin
-          echo ~/.local/bin >> $GITHUB_PATH
           curl -L https://sdk.dfinity.org/downloads/dfx/${{ matrix.sdk_ver }}/x86_64-linux/dfx-${{ matrix.sdk_ver }}.tar.gz | tar xz --strip-components 1 -C ~/.local/bin
 
-      - name: cache
-        uses: actions/cache@v2
-        with:
-          key: 1-cargo-cache-${{ github.run_id }}
-          restore-keys: 1-cargo-cache-
-          path: |
-            ~/.cargo
-            dfinity/rs/target
+      - name: setup-nix
+        uses: cachix/install-nix-action@v16
+
+      - name: setup-nghttpx
+        run: |
+          nix profile install nixpkgs#nghttp2
```

Comment:
I believe there is a `nghttp2` package in the ubuntu-latest that we use for the build. Is there any benefit in installing another package management system?

Reply:
`nghttp2` in ubuntu doesn't work. I've yet to figure out why, but the nix version is sufficient to keep stuff working for now.